70e477cf489ac561c206ef82f74b7d2193128d36
diff --git a/bindings/qt/QTMegaGlobalListener.cpp b/bindings/qt/QTMegaGlobalListener.cpp
index cf8dc2ced..ee843414f 100644
--- a/bindings/qt/QTMegaGlobalListener.cpp
+++ b/bindings/qt/QTMegaGlobalListener.cpp
@@ -63,11 +63,13 @@ void QTMegaGlobalListener::onEvent(MegaApi *api, MegaEvent *e)
     QCoreApplication::postEvent(this, event, INT_MIN);
 }
 
+#ifdef ENABLE_SYNC
 void QTMegaGlobalListener::onGlobalSyncStateChanged(MegaApi *api)
 {
     QTMegaEvent *event = new QTMegaEvent(api, (QEvent::Type)QTMegaEvent::OnGlobalSyncStateChanged);
     QCoreApplication::postEvent(this, event, INT_MIN);
 }
+#endif
 
 void QTMegaGlobalListener::customEvent(QEvent *e)
 {
@@ -92,9 +94,11 @@ void QTMegaGlobalListener::customEvent(QEvent *e)
         case QTMegaEvent::OnEvent:
             if(listener) listener->onEvent(event->getMegaApi(), event->getEvent());
             break;
+#ifdef ENABLE_SYNC            
         case QTMegaEvent::OnGlobalSyncStateChanged:
             if(listener) listener->onGlobalSyncStateChanged(event->getMegaApi());
             break;
+#endif
         default:
             break;
     }
