54522a9ecc63f2f83b61c3bff32a0c2b4e16160d
diff --git a/cmake/modules/sdklib_libraries.cmake b/cmake/modules/sdklib_libraries.cmake
index b50d1b7ca..0941a0217 100644
--- a/cmake/modules/sdklib_libraries.cmake
+++ b/cmake/modules/sdklib_libraries.cmake
@@ -10,7 +10,7 @@ macro(load_sdklib_libraries)
         if(WIN32)
             target_link_libraries(SDKlib PUBLIC unofficial-sodium::sodium)  # TODO: Private for SDK core
         else()
-            target_link_libraries(SDKlib PRIVATE unofficial-sodium::sodium unofficial-sodium::sodium_config_public)
+            target_link_libraries(SDKlib PRIVATE unofficial-sodium::sodium)
         endif()
 
         find_package(unofficial-sqlite3 REQUIRED)
@@ -60,7 +60,7 @@ macro(load_sdklib_libraries)
 
         if(USE_LIBUV)
             find_package(libuv REQUIRED)
-            target_link_libraries(SDKlib PRIVATE $<IF:$<TARGET_EXISTS:uv_a>,uv_a,uv>)
+            target_link_libraries(SDKlib PRIVATE $<IF:$<TARGET_EXISTS:libuv::uv_a>,libuv::uv_a,libuv::uv>)
             set(HAVE_LIBUV 1)
         endif()
 
