60d86e1ae0a686f32dbfc838e7a2162ea240fc5e
diff --git a/CMakeLists.txt b/CMakeLists.txt
index b9a95dc47..a85c1aaa7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,6 +23,7 @@ set(QT_CREATOR_SKIP_VCPKG_SETUP TRUE CACHE BOOL "")
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/cmake/modules)
 
 set(VCPKG_ROOT "" CACHE PATH "If set, it will build and use the VCPKG packages defined in the manifest file")
+set(VCPKG_ROOT "ANY_STRING")
 
 ## Configurable options ##
 include(sdklib_options)
